tests/visitors/test_models.py

```diff
@@ -1,9 +1,221 @@
 from parameterized import parameterized
 
-from django_codemod.visitors.models import ModelsPermalinkTransformer
+from django_codemod.visitors.models import (
+    ModelsPermalinkTransformer,
+    OnDeleteTransformer,
+)
 from tests.visitors.base import BaseVisitorTest
 
 
+class TestOnDeleteTransformer(BaseVisitorTest):
+
+    transformer = OnDeleteTransformer
+
+    def test_foreign_key(self) -> None:
+        pass
```

Comment:
Do you intend to add more thing here or can we remove this empty function?

Reply:
oops! good catch, meant to test another path. I'll put in your non-migration model test in here 👍